project_restorer/src/main/scala/PlutoCoreMessageProcessor.scala

```diff
@@ -19,21 +20,25 @@ import org.slf4j.LoggerFactory
 import java.time.ZonedDateTime
 import scala.concurrent.{ExecutionContext, Future}
 
-class PlutoCoreMessageProcessor(mxsConfig:MatrixStoreConfig)(implicit mat:Materializer,
-                                                             matrixStoreBuilder: MXSConnectionBuilder,
-                                                             vidispineCommunicator: VidispineCommunicator,
-                                                             ec:ExecutionContext) extends MessageProcessor {
+class PlutoCoreMessageProcessor(mxsConfig: MatrixStoreConfig)(implicit mat: Materializer,
+                                                              matrixStoreBuilder: MXSConnectionBuilder,
+                                                              vidispineCommunicator: VidispineCommunicator,
+                                                              ec: ExecutionContext) extends MessageProcessor {
   private val logger = LoggerFactory.getLogger(getClass)
 
   private val statusesMediaNotRequired = List(EntryStatus.Held.toString, EntryStatus.Completed.toString, EntryStatus.Killed.toString)
 
-  def searchAssociatedOnlineMedia(projectId: Int, vidispineCommunicator: VidispineCommunicator): Future[Seq[OnlineOutputMessage]] = {
+  def searchAssociatedOnlineMedia(projectId: Int, vidispineCommunicator: VidispineCommunicator): Future[Seq[OnlineOutputMessage]] =
     onlineFilesByProject(vidispineCommunicator, projectId)
-  }
 
-  def onlineFilesByProject(vidispineCommunicator: VidispineCommunicator, projectId: Int): Future[Seq[OnlineOutputMessage]] = {
+  def onlineFilesByProject(vidispineCommunicator: VidispineCommunicator, projectId: Int): Future[Seq[OnlineOutputMessage]] =
     vidispineCommunicator.getFilesOfProject(projectId)
-      .map(_.map(item => InternalOnlineOutputMessage.toOnlineOutputMessage(item)))
+      .map(_.filter(item => !isBranding(item)).map(item => InternalOnlineOutputMessage.toOnlineOutputMessage(item)))
```

Comment:
Could probably be written more succinctly as `.map(_.filterNot(isBranding))...`

Reply:
Indeed :)